### ISSUE TITLE
suggestion Paste URL 

### ISSUE THREAD

**Issue by mrfragger** [tomasklaen/uosc#979]
I continue using https://github.com/9beach/mpv-config/blob/main/scripts/copy-and-paste.lua
only for two reasons instead of using Ctrl+v in UOSC which works just fine.  
On Mac and Windows it works..Linux one must install xclip.  

UOSC uses the ziggy bins which is great.  Now used not just for downloading opensubtitles. 

First and main reason is you can do it when any media is playing.  In uosc one must open playlist first before pasting.

in copy-and-paste script it just appends it to the playlist
    -- Keybind for append file paths or URLs in clipboard to current track.
    append_to_current_track_keybind = 'Ctrl+v Alt+v Meta+v',

If mpv is just idle it loads the URL and plays.  If watching something then appends to playlist.

2nd reason but this might be better how UOSC has it currently is if there's a playlist with multiple items one must open video track to be able to paste url.  In the copy-and-paste script it'll append it to the end of a long playlist.

In a nutshell...would be nice to Ctrl+v having it append to queue playlist without having to open playlist first

**Comment by Solarunit** [comment]
I wish we had paste (youtube) url menu. With a field with the whole url to confirm. It also can automatically paste url that was copied. And some kind play now button or just play on Enter.

**Comment by po5** [comment]
This would be better suited as an external script (they can use ziggy as well), as you're asking for behavior that's outside of the menu.

@Solarunit Sounds like you just want an "open in mpv" button in your browser, for which there are already programs on Github.

**Comment by Solarunit** [comment]
@po5 I know that such extensions exists but I wouldn't mind a simple input url field in mpv too :slightly_smiling_face:

**Comment by tomasklaen** [comment]
Well, I could provide commands for copying and pasting paths/urls, but I like how explicit and ready to use the current workflow is. If you want to:

- add to playlist -> paste to playlist
- open immediately -> paste to open-file menu
- add as track -> paste into track menu

and you don't need to configure anything, or set up any keybinds, it's already part of the menus.

But it would be nice to st up a default paste behavior for no open menu for people, and since ziggy can already work with clipboard, why not.

**Comment by mrfragger** [comment]
ok I was using uosc/items so that's why I couldn't pull up playlist to paste it.  uosc/playlist solves it..yeah I'd say just do default paste behavior (append to playlist) for no open menu.

**Comment by tomasklaen** [comment]
New commands: `paste`, `paste-to-open`, `paste-to-playlist`, `copy-to-clipboard`

See readme for details.

I also added ability to copy paths of selected items in playlist or file/directory menus.
